console/console-init/ui/src/stories/LinkList.stories.tsx

```diff
@@ -5,10 +5,13 @@
 
 import React from "react";
 import { MemoryRouter } from "react-router";
-import { LinkList, ILink } from "../modules/connection-detail/LinkList";
+import {
+  ConnectionLinksList,
+  ILink
+} from "../modules/connection-detail/components/ConnectionLinksList/ConnectionLinksList";
```

Comment:
i think import path is wrong and don't use relative path ../. It should be 
import {
  ConnectionLinksList,
  ILink
} from "modules/connection-detail/components";